src/app/presentation/home/pages/blogs-pages/blogs-details/blogs-details.component.html

```diff
@@ -1 +1,20 @@
-<p>blogs-details works!</p>
+<div id="container" class="animate__animated animate__fadeIn">
+  <main>
+    <app-basic-header [options]="{redirect: '/home/main-blogs', name:'Tips & Topic Details'}"/>
+    
+    <div class="mt-6 p-4 pb-12">
+      @if (blog()) {
+        <img class="w-full aspect-video rounded-md" [src]="blog()?.images?.[0]" [alt]="blog()?.title">
+        <p class="text-gray-500 dark:text-gray-200 mt-4"> {{blog()?.description}} </p>
+      } @else {
+        <app-square-skeleton [dimensions]="{width: '100%', heigth: '300px'}" />
+
+        @for (p of [1,2,3,4,5,6,7,8,9]; track $index) {
```

Comment:
Una cosa dejarlo asi seteado con 0,1,2,3,4,5,6,7,8,9 no esta muy seteado? no seria mejor hacer un key of keys o algo asi?